pkgs/applications/misc/openlp/default.nix

```
@@ -0,0 +1,76 @@
+# This file contains all runtime glue: Bindings to optional runtime dependencies
+# for pdfSupport, presentationSupport, and media playback.
+{ stdenv, mkDerivation, wrapGAppsHook, python3Packages
+
+# qt deps
+, qtbase, qttools, qtmultimedia
+
+# optional deps
+, pdfSupport ? false, mupdf ? null, ghostscript ? null
+, presentationSupport ? false, libreoffice-unwrapped ? null, enableVlc ? false
+, gstreamerSupport ? false, gst_all_1 ? null, gstPlugins ? (gst: [
```

```suggestion
, pdfSupport ? false, mupdf, ghostscript
, presentationSupport ? false, libreoffice-unwrapped
, enableVlc ? false
, gstreamerSupport ? false, gst_all_1, gstPlugins ? (gst: [
```

``? null`` messes with overriding and can break it.

---

I removed ghostscript and all `null`s, and added a real check for a non-empty gstPlugin list in 660a3abb0850535093bff14b1c8bca56a6ef8cb1. I also renamed `enableVlc` to `vlcSupport` to be consistent.